datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -749,6 +750,18 @@ type GlossaryTerm implements Entity {
     Edges extending from this entity
     """
     relationships(input: RelationshipsInput!): EntityRelationshipsResult
+
+    
+    """
+    Schema metadata of the dataset
+    """
+    schema: Schema @deprecated(reason: "Use `schemaMetadata`")
```

Comment:
lets not add the deprecated one- schemaMetadata is fine. lets remove this line

Reply:
deleted